numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2541,6 +2542,127 @@ def add_newdoc(place, name, doc):
 
     """)
 
+add_newdoc('numpy.core.umath', 'matmul',
+    """
+    Matrix product of two arrays.
+
+    Parameters
+    ----------
+    x1, x2 : array_like
+        Input arrays, scalars not allowed.
+    out : ndarray, optional
+        Output argument. This must have the exact kind that would be returned
+        if it was not used. In particular, it must be C-contiguous and have
+        both the right type and the right dtype. This is a performance feature.
+        Therefore, if these conditions are not met, an exception is raised,
+        instead of attempting to be flexible.
+    **kwargs
```

Comment:
This won't render correctly. Either ommit the `**` or use a varriant of `\\*\\*kwargs : Kwargs`.

Reply:
the doc [looks OK](https://4868-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.matmul.html) to me, just like the other ufuncs